docs/tutorials/multimodal/object_detection/detection_fast_finetune_coco.md

```diff
@@ -0,0 +1,165 @@
+# AutoMM Detection - Fast Finetune on COCO Format Dataset
+:label:`sec_automm_detection_fast_finetune_coco`
+
+In this section, our goal is to fast finetune a pretrained model on VOC2017 training set, 
+and evaluate it in VOC2007 test set. Both training and test sets are in COCO format.
+See :ref:`sec_automm_detection_prepare_voc` for how to prepare VOC dataset,
+and :ref:`sec_automm_detection_convert_to_coco` for how to convert other datasets to COCO format.
+
+To start, let's import MultiModalPredictor:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+
+We select the YOLOv3 with MobileNetV2 as backbone,
+and input resolution is 320x320, pretrained on COCO dataset. With this setting, it is fast to finetune or inference,
+and easy to deploy.
+For more model choices, see :label:`sec_automm_detection_selecting_models`.
+While using COCO format dataset, the input is the json annotation file of the dataset split.
+In this example, `voc12_train.json` is the annotation file of train split of VOC2012 dataset,
+and `voc07_test.json` is the annotation file of train split of VOC2007 dataset.
+And we use all the GPUs (if any):
+
+```python
+checkpoint_name = "yolov3_mobilenetv2_320_300e_coco"
+num_gpus = -1  # use all GPUs
+
+train_path = "./VOCdevkit/VOCCOCO/voc12_train.json"
+test_path = "./VOCdevkit/VOCCOCO/voc07_test.json"
+```
+
+We create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We need to specify the problem_type to `"object_detection"`,
+and also provide a `sample_data_path` for the predictor to infer the catgories of the dataset.
+Here we provide the `training_path`, and it also works using any other split of this dataset.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    pipeline="object_detection",
```

Comment:
In the text, you mentioned problem_type, but here is still pipeline, we need to fix this later (probably not now).

Reply:
Yes, I'll replace all the `pipeline` to problem type once Xingjian's PR is merged